package.json

```diff
@@ -85,16 +82,17 @@
     "ws": "^7.5.5"
   },
   "devDependencies": {
-    "eslint": "^8.17.0",
+    "audit-ci": "^6.3.0",
+    "eslint": "^8.20.0",
```

Comment:
Why do we have `eslint` here if we are using `standard`?

Reply:
Done